### ISSUE TITLE
{Linked:Bug11089142} [Adaptive Cards > Designer] - Visual Asterisk is not provided for the Expense purpose multi line edit field and update CC/IO edit field. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8698]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H3(OS Build 25951.1010)
Browser Version: Edge Dev 117.0.2033.0
Screen Reader: Narrator

**Pre-Requisite:**
1. Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Navigate the screen and observe the issue.

**Actual Result:**
Visual Asterisk is not provided for the Expense purpose multi line edit field and update CC/IO edit field.

**Observation:**
Narrator is announcing as 'Expense Purpose edit Ground Transportation: UBER SAN FRANCISCO | Mon, Jun 19, 2023 required and enter a CC/IO edit 10257098 required 10257098'.

**Expected Result:**
Visual Asterisk should be provided for the Expense purpose multi line edit field and update CC/IO edit field.

**User Impact:**
Cognitive Users are getting impacted as (Asterisk) is not defined for the required fields. If Star (Asterisk) symbol is not provided users may not fill the required edit fields as it is not mandatory.

**WCAG Reference:**

[Understanding Success Criterion 1.3.1: Info and Relationships (w3.org)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/66f0044a-0462-4b1d-9e6b-8c968ff5e848

![WCAG1 3 1-Visual Asterisk is not provided for the Expense purpose multi line edit field and update CC IO edit field](https://github.com/microsoft/AdaptiveCards/assets/95913748/39f4f07c-525b-4cf9-b044-edb24dc1119c)

[Test JSON.docx](https://github.com/microsoft/AdaptiveCards/files/12817006/Test.JSON.docx)

### Card JSON

```JSON
Attached in the screenshot section.
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

The visual asterisk is not present on this card because the `Input.Text` `label` property is not being used. If the label property is used instead of a separate TextBlock, the asterisk will appear.

![image](https://github.com/microsoft/AdaptiveCards/assets/98650930/a7920f19-9ffb-4717-af3a-482677048431)

Let me know if this doesn't work for you!
Anna

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the comment above.
